wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/service/konfiguriertewahltage/KonfigurierteWahltageServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package de.muenchen.oss.wahllokalsystem.adminservice.service.konfiguriertewahltage;
+
+import de.muenchen.oss.wahllokalsystem.adminservice.service.common.KonfigurierterWahltagModel;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.common.WahlenClient;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.konfigurierterwahltag.KonfigurierteWahltageService;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.konfigurierterwahltag.KonfigurierterWahltagValidator;
+import de.muenchen.oss.wahllokalsystem.adminservice.service.wahltermindaten.KonfigurierterWahltagClient;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.TechnischeWlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import java.time.LocalDate;
```

Comment:
Da die entsprechende Objektvariable unused ist.

```suggestion
import de.muenchen.oss.wahllokalsystem.wls.common.exception.TechnischeWlsException;
import java.time.LocalDate;
```